src/makeCalculator.js

```diff
@@ -5,6 +5,41 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    methods: [],
+  };
+
+  calculator.reset = () => {
+    calculator.result = 0;
+
+    return calculator;
+  };
+
+  calculator.add = (number) => (calculator.result += number);
+  calculator.subtract = (number) => (calculator.result -= number);
+  calculator.multiply = (number) => (calculator.result *= number);
+
+  calculator.divide = (number) => {
+    if (number !== 0) {
+      calculator.result /= number;
+    }
+  };
```

Comment:
The `divide` method currently checks if the number is not zero before dividing. However, it would be better to handle the case where `number` is zero more explicitly, such as by returning an error message or a specific value to indicate the operation was not successful.